src/asciidoc/docs/about/contribute.adoc

```diff
@@ -0,0 +1,176 @@
+
+Anyone can open an issue about documentation, or contribute a change with a pull request to the `madewithkoji/docs` https://github.com/madewithkoji/docs[GitHub repository].
+
+link to code of conduct?
+
+include::../_includes/tip-discord-server.adoc[tags=*]
+
+== Editing the documentation
+
+The documentation is written in https://asciidoc.org/[Asciidoc] which is a lightweight markup language for simplifying the creation of documentation.
+All the AsciiDoc files that map to this site can be found in the repository under `src/asciidoc/`.
+
+Here's a quick example:
+
+[source, asciidoc]
+----
+= Koji for Developers
+Create your own templates for https://withkoji.com[Koji].
+
+
+== Use any framework
+
+* React
+* Vanilla JS
+
+[source,javascript]
+console.log('Hello World');
+----
+
+
+To see more and get a better understanding of the syntax visit the https://powerman.name/doc/asciidoc[AsciiDoc cheatsheet] and the https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[AsciiDoc Syntax Quick Reference].
+
+We also have are a few extensions on AsciiDoc syntax added in to make the developer site ever better.
+
+=== Page Attributes
+Page attributes are used to add extra data related to a document.
+Add these to the top of your document, just under the document title.
+
+Page slug::
+	This corresponds to the page url and is used for linking between different pages. +
+	`:page-slug: example-page`
+Page description::
+	This is a short description of the content of the page and is used when sharing the link on social media and chat platforms. +
+	`:page-description: This page is an example`
+Page banner::
+	This references an image which is used as the thumbnail when sharing the link on social media and chat platforms. +
+	`:page-banner: \{imagesDir\}/example-page.png`
+
+NOTE: `\{imagesDir\}` refers to the image directory, which can be found in the repository at `static/images`
+
+TIP: If you have block images on your page you don't need to set the page banner, the first image will be used instead
+
+=== Tabs
+Sometimes it can be useful to sort your content into tabs.
+You'll see them used particularly often to display code snippets for differing frameworks.
+To add tabs into your document use the following syntax:
+
+[source, asciidoc]
+----
+[.tabs]
+--
+
+.Tab 1 title
+Tab 1 content.
+
+.Tab 2 title
+Tab 2 content.
+
+--
+----
+
+The above code will output the following:
+
+[.tabs]
+--
+
+.Tab 1 title
+Tab 1 content.
+
+.Tab 2 title
+Tab 2 content.
+
+--
+
+You will also notice that on some pages changing tabs will switch the tabs in multiple places (e.g. choosing a coding framework in a blueprint article). You can use scope attribute to sync multiple sets of tabs with one another, by setting two sets of tabs with the same scope attributes, whenever a tab is selected all the tabs with the same name will also be selected across matching scope names.
```

Comment:
```suggestion
You will also notice that on some pages changing tabs will switch the tabs in multiple places (for example, choosing a coding framework in a blueprint article). 
To sync multiple sets of tabs, use the `scope` attribute and matching tab titles. Then, selecting one tab on the page also selects any tabs with the same scope and the same title.
```
This paragraph is a little confusing. Think the suggestion needs some word-smithing, too, but hopefully gives you some ideas